### PR TITLE
[Storage] Fixed cspell typos in storage-datalake

### PR DESCRIPTION
**Pull request by wonhyeongseo**
# Description

fix https://github.com/Azure/azure-sdk-for-python/issues/22677

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

File: .vscode/cspell.json [hunk 1]
```diff
@@ -504,6 +504,29 @@
...
+        "mypathfilesystem", "myclientfilesystem", "mydirectoryfilesystem", "myfilesystemforgetclient",
+        "mynewdirectory", "mydirectory", "mynewfile", "myfilesystem", "mynewfilesystem",
+        "myfilesystemforlistpaths", "cpkfile", "dont", "fsname", "fpath", "dname", "racwdlmeop",
+        "filesys", "nonexistentfs", "testid", "cpkdirectory", "cpksubdirectory"
+      ]
+    },
+    {
+      "filename": "sdk/storage/azure-storage-file-datalake/**/*.md",
+      "words": ["thead", "tbody", "myfilesystem"]
+    },
```
Comment (jalauzon-msft) [inline .vscode/cspell.json:525]: Could you combine all the datalake ones into one section `"sdk/storage/azure-storage-file-datalake/**"`.

Can you also do one word per line to match the style of the rest of the files.

File: .vscode/cspell.json [hunk 2]
```diff
@@ -504,6 +504,29 @@
...
+    {
+      "filename": "sdk/storage/**/_shared/*.py",
+      "words": ["yarl", "mday"]
```
Comment (jalauzon-msft) [inline .vscode/cspell.json:528]: I think after #25923 you shouldn't need to add anything for `_shared`. Please merge from `main` and see if that eliminates the need for this.

**Review by jalauzon-msft — COMMENTED** [review]
Thanks @wonhyeongseo! Just two minor comments. Sorry for the delay getting to this review as well.

File: .vscode/cspell.json (hunk 2 above)
Comment (wonhyeongseo) [reply .vscode/cspell.json]: Yes, my friend @KdHyeon0661 has done a great job!

File: .vscode/cspell.json (hunk 1 above)
Comment (wonhyeongseo) [reply .vscode/cspell.json]: Hello @jalauzon-msft ! I have reordered the words into one section as you requested. Thank you so much for your work :)

**Comment by wonhyeongseo** [comment]
No worries, I am happy to contribute 😄

**Review by jalauzon-msft — APPROVED** [review]
Looks good now! Thanks again @wonhyeongseo!

File: .vscode/cspell.json (hunk 1 above)
Comment (jalauzon-msft) [reply .vscode/cspell.json]: Thanks! Looks good.

**Pull request merged**